release.json

```diff
@@ -7,11 +7,11 @@
     },
     "nightly": {
         "INTEGRATIONS_CORE_VERSION": "7.53.x",
-        "OMNIBUS_SOFTWARE_VERSION": "7.53.x",
-        "OMNIBUS_RUBY_VERSION": "7.53.x",
+        "OMNIBUS_SOFTWARE_VERSION": "375618d70253293d71b13f9385260aa3dedd7125",
+        "OMNIBUS_RUBY_VERSION": "bc5e0fb395c4b106967f97f8738d23705e5bc933",
         "JMXFETCH_VERSION": "0.49.0",
         "JMXFETCH_HASH": "b5c2c3ff27603f469bb11961d559f1154887963e02b9d70d5f1fc7efa527a486",
-        "MACOS_BUILD_VERSION": "7.53.x",
+        "MACOS_BUILD_VERSION": "nschweitzer/master",
```

Comment:
We don't support macos in agent6, so I fixed this temporarily but will decommission macos later.